src/config/unit_test.rs

```diff
@@ -45,7 +45,11 @@ async fn build_unit_tests(builder: ConfigBuilder) -> Result<Vec<UnitTest>, Vec<S
         match build_unit_test(test, &config).await {
             Ok(t) => tests.push(t),
             Err(errs) => {
-                let mut test_err = errs.join("\n");
+                let mut test_err = errs
```

Comment:
Generally it's not good practice to turn errors into strings, but here the caller returns strings, so it's warranted.